### DISCUSSION TITLE
Mobile version still in version 7.13.2

### DISCUSSION THREAD

**Discussion by emfcamara** [super-productivity/super-productivity#2847] [Q&A]
Hi, Like the title says the mobile version is still 7.13.2 on my phone and it says it's the latest but the latest version is 7.14.3 and that I use on the desktop. Any idea what's going on? Thank you for any help.

**Comment by johannesjo** [comment]
Thanks for letting me know! I will make an update soon. Seems like the auto deployment for mobile didn't work..
